otherlibs/configurator/test/blackbox-tests/pkg-config-args.t/dune

```diff
@@ -1,11 +1,11 @@
 (executable
- (name pkg_config)
- (modules pkg_config))
+ (name pkgconf)
+ (modules pkgconf))
```

Comment:
It would be nice if both dummy `pkg_config` and `pkgconf` were both built to be able to demonstrate the fallback behavior in tests.

Reply:
I put it some thought even before your remark and that fallback is subtle to demonstrate as it requires to remove the `pkgconf` you have on your system from your PATH else it is favored over the local `pkg-config`...
If you figure out a way, I can implement it :)